jax/scipy/linalg.py

```diff
@@ -238,18 +237,31 @@ def triu(m, k=0):
 _expm_description = textwrap.dedent("""
 In addition to the original NumPy argument(s) listed below,
 also supports the optional boolean argument ``upper_triangular``
-to specify whether the ``A`` matrix is upper triangular.
+to specify whether the ``A`` matrix is upper triangular, and the optional
+argument ``max_n_squarings`` to specify the max number of squarings allowed
+in the scaling-and-squaring approximation method. Return nan if the actual
+number of squarings required is more than ``max_n_squarings``.
```

Comment:
It would be nice to add a reference on appropriate values of `max_n_squarings`, or at least some guidance on why you would want a bigger number (e.g., for larger matrix norm). You could even note the formula, e.g., `n_squarings = max(0, floor(log2(sum(abs(A))) - c))` with `c=2.43` for float64 and `c=1.97` for float32.